databricks/koalas/indexes.py

```diff
@@ -517,6 +585,88 @@ def copy(self):
         result = MultiIndex(ks.DataFrame(internal))
         return result
 
+    def symmetric_difference(self, other, result_name=None, sort=None):
```

Comment:
The logic is exactly the same as of `Index`. I don't think we need this with the fix mentioned above.

Reply:
@ueshin , thanks for the review! that sounds reasonable. i'll fix them 👍